graylog2-web-interface/src/components/common/DataTable.jsx

```diff
@@ -29,7 +29,7 @@ class DataTable extends React.Component {
     /** Label to use next to the data filter input. */
     filterLabel: PropTypes.string,
     /** List of object keys to use as filter in the data filter input. Use an empty array to disable data filter. */
-    filterKeys: PropTypes.array.isRequired,
+    filterKeys: PropTypes.array,
```

Comment:
Multiple places throughout the app we just pass an empty array to `filterKeys`. This just makes it a default prop so we don't get console warnings in areas where we miss the empty array.